### PR TITLE
Review

### PR DESCRIPTION
**Pull request by lukasniemeier-zalando**
review me

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -28,12 +28,13 @@ app.all('*', function(request, response) {
...
     }
 
-    var payload = JSON.parse(request.body.payload.replace(/\\"/g, '"'));
+    var payload = JSON.parse(request.body.payload);
 
     console.log(type + ' @ ' + room);
 
     var template = templates.getTemplate(type);
     if (!template) {
+        console.log("Unknown event type " + type);
```
Comment (lukasniemeier-zalando) [inline index.js]: please remove the logging

File: index.js [hunk 2]
```diff
@@ -45,7 +46,10 @@ app.all('*', function(request, response) {
             auth: { bearer: token }
         },
         function (error, res, b) {
-            response.status(res.statusCode).send(b);
+            if (error) {
+                console.log("Error forwarding event: " + error);
```
Comment (lukasniemeier-zalando) [inline index.js]: foobar blub

File: index.js (hunk 1 above)
Comment (lukasniemeier) [inline index.js]: are you sure?

File: index.js (hunk 2 above)
Comment (lukasniemeier) [inline index.js]: rabums

**Review by lukasniemeier — APPROVED** [review]
All fine.

**Review by lukasniemeier-zalando — COMMENTED** [review]
abc

**Review by lukasniemeier-zalando — COMMENTED** [review]
ok

**Pull request closed without merging**